src/main/resources/config/liquibase/changelog/00000000000000_initial_schema.xml

```diff
@@ -46,6 +46,132 @@
             <column name="last_modified_date" type="timestamp"/>
         </createTable>
 
+<!--        Address-->
+
+        <createTable tableName="address_state">
+            <column name="id" type="bigint" >
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="state" type="varchar(20)" />
+        </createTable>
+
+        <loadData tableName="address_state" file="config/liquibase/data/address_state.csv" separator=";">
+            <column name="id" type="numeric" />
+            <column name="state" type="string" />
+        </loadData>
+
+        <createTable tableName="address">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="street_address" type="varchar(50)" />
+            <column name="unit" type="varchar(20)" />
+            <column name="city" type="varchar(20)" />
+            <column name="state" type="bigint" />
+            <column name="zip_code" type="varchar(10)" />
+            <column name="user_id" type="bigint" />
+        </createTable>
+
+        <addForeignKeyConstraint baseTableName="address" baseColumnNames="state" constraintName="state_id" referencedTableName="address_state"
+                                 referencedColumnNames="id"/>
+
+<!--        Game/Product Status-->
+
+        <createTable tableName="product_status">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false" />
+            </column>
+            <column name="status" type="varchar(30)" />
+        </createTable>
+
+        <loadData tableName="product_status" file="config/liquibase/data/product_status.csv" separator=";">
+            <column name="id" type="numeric" />
+            <column name="status" type="string" />
+        </loadData>
+
+        <createTable tableName="game">
+            <column name="id" type="varchar(15)">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="year_published" type="smallint" />
+            <column name="min_players" type="smallint" />
+            <column name="max_players" type="smallint" />
+            <column name="min_age" type="tinyint" />
+            <column name="name" type="varchar(200)" />
```

Comment:
I know this is silly, but can we move the name field to right after id so when we are looking in the DB they are right next to each other?

Reply:
Sure! np 👍